### ISSUE TITLE
There is a duplicate column in Supplementary Table S1. is it N. of negative samples?

### ISSUE THREAD

**Issue by aifact** [liulifenyf/Arabidopsis.tf.pred#5]
![image](https://user-images.githubusercontent.com/79238945/117448191-05e52280-af71-11eb-92cc-382b09323fb2.png)

**Comment by liulifenyf** [comment]
Yes, you're right. Thank you for your remind. I'll rectify it immediately!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "liulifenyf/TSPTFBS"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月7日(星期五) 晚上8:18
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[liulifenyf/TSPTFBS] There is a duplicate column in Supplementary Table S1. is it N. of negative samples? (#5)

**Comment by liulifenyf** [comment]
Oh， it's a supplementary table, i can not change it anymore. But you are right, next time we'll pay more attention about it. Thank you!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "liulifenyf/TSPTFBS"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月7日(星期五) 晚上8:18
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[liulifenyf/TSPTFBS] There is a duplicate column in Supplementary Table S1. is it N. of negative samples? (#5)

**Comment by aifact** [comment]
It doesn't matter. This doesn't change the key conclusion in your paper. By the way, does this work generalize TFs supported by experiments to those TFs not studied?

 I know this work can search TFs binding site sqe. by a fast experiment. https://mp.weixin.qq.com/s/slTosggaCEWwUiZ68LzKtw
